getting_help/README.md

```diff
@@ -1,12 +1,7 @@
 # Getting help
 
-Join us in Slack:
-
-<script async defer src="https://pact-slack.herokuapp.com/slackin.js"></script>
```

Comment:
This is the slack script I removed.

Reply:
could replace it with the SVG (the heroku link should still work, but perhaps gitbook doesn't like the `<script>` element:

```
[![slack](http://slack.pact.io/badge.svg)](http://slack.pact.io)
```
or
```
<a href="http://slack.pact.io"><img src="http://slack.pact.io/badge.svg"></a>
```

Reply:
Aha! Thanks. I've put that in.